### PR TITLE
refs task/8152: Edit migration

### PR DESCRIPTION
**Pull request by dinhtuan242**
Thêm bảng Unit
https://edu-redmine.framgia.vn/issues/8152

### CODE REVIEW

**Comment by dinhtuan242** [comment]
ready

File: resources/views/fontend/homepages/property_list.blade.php [hunk 1]
```diff
@@ -61,7 +61,7 @@
                                     </h1>
                                     <div class="location">
                                         <a href="#">
-                                            <i class="fa fa-map-marker"></i>{{ $property->districts->name }}
+                                            <i class="fa fa-map-marker"></i>{{ $property->districts->name ?? ''}}
```
Comment (BunnyPi04) [inline resources/views/fontend/homepages/property_list.blade.php]: convention

**Comment by dinhtuan242** [comment]
ready

File: database/seeds/UnitTableSeeder.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('units')->insert([
+            ['name' =>'Thỏa thuận'],
+            ['name' => 'Triệu'],
+            ['name' => 'Tỷ'],
+            ['name' => 'Trăm nghìn/m2'],
+            ['name'=> 'Trăm nghìn/tháng'],
```
Comment (BunnyPi04) [inline database/seeds/UnitTableSeeder.php]: convention

**Comment by dinhtuan242** [comment]
ready

**Pull request merged**